dbDataGenAndSeeding/schemaOutline.json

```diff
@@ -0,0 +1,24 @@
+{
+  "photo": {
+    "imageId": "number auto inc",
+    "businessId": "number/foreign key business",
+    "imageUploaderId": "number/foreign key users",
+    "imageUrl": "string",
+    "imageUploadDate": "date",
+    "imageComment": "string",
+    "helpfulCount": "number",
+    "reported": "boolean"
+  },
+  "user": {
+    "userId": "number auto inc",
+    "userName": "string",
+    "profileImageUrl": "string",
+    "friendsCount": "number",
+    "reviewsCount": "number",
+    "eliteStatus": "boolean"
+  },
+  "business": {
+    "businessId": "number auto inc",
+    "businessName": "string"
+  }
+}
```

Comment:
I think i might have missed something from the requirements if this was one. Regardless, this is a good representation of the data to be received via request.

Reply:
I actually set this up because I initially thought I would be using mongo. But I kept it after switching to mysql since it was a good reference for what I want my server rendered json to look like.